ci/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Setup configuration
+cp ci/testsuite.cfg.$DB testsuite.cfg
+sudo cp ci/haas.cfg.$DB /etc/haas.cfg
+sudo chown travis:travis /etc/haas.cfg
+
+# Database Setup
+if [ $DB = postgres ]; then
+    psql --version
+    psql -c 'CREATE DATABASE haas_tests;' -U postgres
+    psql -c 'CREATE DATABASE haas;' -U postgres
+elif [ $DB = sqlite ]; then
+    # Workaround to create an empty sqlite db file
```

Comment:
What are the warnings?

Generally speaking, when you have something like this, you should be precise about the details of the problem it's working around. Too easy to get into a situation where you're looking at code that claims it fixes *something,* and you don't know what.